site/src/scss/_base.scss

```diff
@@ -21,9 +22,21 @@ pre code {
   font-weight: bold;
   padding: rem-calc(20) rem-calc(64);
 
+  @include breakpoint(large) {
+    min-width: rem-calc(291);
+  }
+
   &:hover {
     border-style: solid;
     border-color:$buttonblue;
     box-shadow: 0 9px 5px 0 rgba(0, 0, 0, 0.15)
   }
+
+  .icon {
```

Comment:
Woah, I don't think we want absolute positioning on every icon, do we?  That really limits our ability to resize these things and have them flow nicely.